### PR TITLE
Don't include card images in JAR files.

### PR DESCRIPTION
**Pull request by jwarwick**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -76,9 +76,12 @@
   :aot [#"game\.*"
         #"web\.*"
         #"tasks.fetch"
-        #"jinteki\.*"]
+        #"jinteki\.*"
+        #"web.core"]
```
Comment (NoahTheDuke) [inline project.clj:80]: Isn't this handled by the `#"web\.*"` inclusion above?

Comment (jwarwick) [reply project.clj:80]: Sadly, it still generates a warning message if the `main` function isn't called out explicitly.

Comment (jwarwick) [reply project.clj:80]: But, yes, it was being included in the AOT from the regex.

Comment (NoahTheDuke) [reply project.clj:80]: Oh hell yeah, that's annoyed me forever.

**Pull request merged**